lib/vet360/models/email.rb

```diff
@@ -17,6 +17,19 @@ class Email < Base
         length: { maximum: 255, minimum: 6 }
       )
 
+      def to_request()
+        {
+          bio: {
+            emailAddressText: @email_address,
+            emailId: @id,
+            # emailPermInd: true, # @TODO ??
+            originatingSourceSystem: Settings.vet360.cuf_system_name,
+            sourceDate: @source_date,
+            vet360Id: 1 # current_user.vet360_id # @TODO
```

Comment:
I would parameterize this value so we can pass in the `vet360_id` of a user from the service class.